### DISCUSSION TITLE
How to seed data according to environment variables on v3.0.1

### DISCUSSION THREAD

**Discussion by shimopino** [prisma/prisma#9327] [Q&A] [answered]
Prior to v3.0.1, I used environment variables to control where input data seeds, as shown below.

```json
"scripts": {
  "seed:dev": "dotenv -e .env.dev -- prisma db seed --preview-feature",
  "seed:test": "dotenv -e .env.test -- prisma db seed --preview-feature"
}
```

However, since the version v3.0.1, we have to use the single command to submit data as shown below.

```json
"prisma": {
  "seed": "ts-node prisma/seed.ts"
}
```

How can I change the destination of the data depending on the environment variables as in the previous version?

this is sample output

```bach
❯❯❯ npx dotenv -e .env.dev -- npx prisma db seed --preview-feature
prisma:warn Prisma "db seed" was in Preview and is now Generally Available.
You can now remove the --preview-feature flag.
Error: To configure seeding in your project you need to add a "prisma.seed" property in your package.json with the command to execute it:

1. Open the package.json of your project
2. Add the following example to it:

"prisma": {
  "seed": "ts-node prisma/seed.ts"
}


3. Install the required dependencies by running:
npm i -D ts-node typescript @types/node

More information in our documentation:
https://pris.ly/d/seeding
```

**Comment by shimopino** [comment, marked as answer]
I was able to solve this problem on my own.

By simply combining the above two commands, I was able to reproduce the expected behavior!

```json
"prisma": {
  "seed": "ts-node prisma/seed.ts"
},
"scripts": {
  "seed:dev": "dotenv -e .env.dev -- prisma db seed",
  "seed:test": "dotenv -e .env.test -- prisma db seed"
}
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
